src/app/kbv/fields.js

```diff
@@ -38,6 +38,9 @@ module.exports = {
   [APP.QUESTION_KEYS.ITA_BANKACCOUNT]: {
     type: "text",
     validate: ["required", "numeric", { type: "exactlength", arguments: [4] }],
+    // Remove maxlength attribute that gets automatically added for exactlength validators
+    // https://github.com/HMPO/hmpo-components/blob/01473508284e1d06ebc72585f970f25f17fba49d/components/hmpo-text/macro.njk#L32
+    attributes: { "maxlength": undefined },
```

Comment:
Setting things to `undefined` might look a little weird, but setting things this way in user space is the way to be overriding these properties 👍 

https://github.com/HMPO/hmpo-components/blob/master/lib/globals.js#L83-L84 shows that the `hmpoGetAttributes` will do a merge of a number of different sources of attributes, and then https://github.com/HMPO/hmpo-components/blob/master/components/hmpo-date/macro.njk#L21-L25 shows those values being overwritten where not previously defined when trying to build up the individual form element properties.

So a present key with any value should be ignored, even if it's undefined.